ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailList.jsx

```diff
@@ -22,14 +24,23 @@ class UpdatesWizardDetailList extends Component {
             <Loader />
           </div>
         ) : (
-          _.map(selectedHardwares, item => (
-            <UpdatesWizardDetailListItem
-              key={item.name}
-              item={item}
-              wizardData={wizardData}
-              onStep2DataSelect={onStep2DataSelect}
-            />
-          ))
+          <>
+            {_.map(selectedHardwares, item => (
+              <UpdatesWizardDetailListItem
+                key={item.name}
+                item={item}
+                wizardData={wizardData}
+                onStep2DataSelect={onStep2DataSelect}
+              />
+            ))}
+            {selectedHardwares.length === 1 && (
```

Comment:
It should be not for selectedHardwares but for SOFTWARES (packages).

Reply:
in step1 it is checking hardware ids.
Will follow the same pattern here.

Reply:
ah, my mistake, will check with software